blocks/callout/callout.js

```diff
@@ -0,0 +1,69 @@
+export default async function decorate(block) {
+  const calloutData = {
```

Comment:
I really like the solution of creating an object with the table data and using that to iterate over; it's super readable.